packages/playwright-test/src/reporters/merge.ts

```diff
@@ -57,18 +72,21 @@ function parseEvents(reportJsonl: Buffer): JsonEvent[] {
   return reportJsonl.toString().split('\n').filter(line => line.length).map(line => JSON.parse(line)) as JsonEvent[];
 }
 
-async function extractAndParseReports(dir: string, shardFiles: string[]): Promise<{ metadata: BlobReportMetadata, parsedEvents: JsonEvent[] }[]> {
+async function extractAndParseReports(dir: string, shardFiles: string[], printStatus: StatusCallback): Promise<{ metadata: BlobReportMetadata, reportFile: string, parsedEvents: JsonEvent[] }[]> {
   const shardEvents = [];
   await fs.promises.mkdir(path.join(dir, 'resources'), { recursive: true });
   for (const file of shardFiles) {
+    printStatus(`extracting: ${file}`);
     const zipFile = new ZipFile(path.join(dir, file));
     const entryNames = await zipFile.entries();
     for (const entryName of entryNames) {
+      printStatus(`extracting: ${file} > ${entryName}`);
       const content = await zipFile.read(entryName);
       if (entryName.endsWith('.jsonl')) {
         const parsedEvents = parseEvents(content);
         shardEvents.push({
           metadata: findMetadata(parsedEvents, file),
+          reportFile: entryName,
```

Comment:
I think it's better to put `zipFile` here, because that's what the user sees.

Reply:
`merging: blah.zip` looked a bit odd, but we can do that